src/renderer/components/ScreenSharePicker.tsx

```diff
@@ -59,6 +59,21 @@ addPatch({
                 match: /this.localWant=/,
                 replace: "$self.patchStreamQuality(this);$&"
             }
+        },
+        {
+            find: "x-google-max-bitrate",
+            replacement: {
+                // eslint-disable-next-line no-useless-escape
+                match: /"x-google-max-bitrate=".concat\(\i\)/,
+                replace: '"x-google-max-bitrate=".concat("80_000")'
+            }
+        },
+        {
+            find: "x-google-max-bitrate",
```

Comment:
replacement takes either one replacement or an array of replacements

instead of having two patches with the same find, merge them into one patch